docs2/site/docs/migrations/migration5.md

```diff
@@ -448,3 +467,33 @@ the managed heap at all.
 `InputObjectGraphType<TSourceType>.ToAST` and `InputObjectGraphType<TSourceType>.IsValidDefault`
 methods were changed in such a way that now you may be required to also override `ToAST` if you override
 `ParseDictionary`. Changes in those methods are made for earlier error detection and schema printing.
+
+### 22. `AutoRegisteringObjectGraphType` changes
+
+The protected method `GetRegisteredProperties` has been renamed to `GetRegisteredMembers`
+and now supports properties, methods and fields, although fields are not included
+with the default implementation. Override the method in a derived class to include fields.
+
+New for v5, methods are included by default. To revert to v4 behavior, which does not
+include methods, create a derived class as follows:
+
+```csharp
+public class AutoRegisteringObjectGraphTypeWithoutMethods<T> : AutoRegisteringObjectGraphType<T>
+{
+    public AutoRegisteringObjectGraphTypeWithoutMethods() : base() { }
+    public AutoRegisteringObjectGraphTypeWithoutMethods(params Expression<Func<T, object?>>[]? excludedProperties) : base(excludedProperties) { }
```

Comment:
A little inaccuracy here. `AutoRegisteringInputObjectGraphType` may work with properties and fields, `AutoRegisteringObjectGraphType` may work with properties, fields and method, but both contain parameter to exclude  only properties. Is it right?

Reply:
You are correct; I suggest we mark that constructor obsolete. It’s not like it can be used by DI (very easily). Or we can just leave it.

Note that it is impractical to use it to exclude methods, since the expression would need a full method call, but it would make sense for fields - although fields require a derived class to be useful. So I didn’t bother to change the functionality here. 

What is inaccurate? I’ll have to reread the documentation when I’m at a pc again.

Reply:
> Or we can just leave it.

OK.

> What is inaccurate?

Expose option to ignore only properties, but OK.